src/preferences/dialog/dlgprefdeckdlg.ui

```diff
@@ -183,6 +183,16 @@ CUP mode:
           </attribute>
          </widget>
         </item>
+		<item>
+		 <widget class="QRadioButton" name="radioButtonBeatsUntilNextCueAndRemaining">
+		  <property name="text">
+		   <string>Beats until next cue and Remaining</string>
+		  </property>
+		  <attribute name="buttonGroup">
+		   <string notr="true">buttonGroupTrackTime</string>
+		  </attribute>
+		</widget>
+		</item>
```

Comment:
please fix the indentation here, and also check if this needs to be added to the TabStops section at the end of the file (or if the `buttonGroup` attribute already does the trick

Reply:
Good catch ! fixing right away, thanks !

Reply:
VS messed with the tabs, will issue another commit, it didnt look like that in the IDE :/